README.md

```diff
@@ -125,6 +125,7 @@ You can deploy this project using your own deployment solution
 👤 **kerlos bahaa**
 
 - GitHub: [@kerlos bahaa ](https://github.com/kerlos-bahaa)
+- Twitter: [@kerlos bahaa](https://twitter.com/KerlosBahaa)
 - LinkedIn: [@kerlos bahaa](https://www.linkedin.com/in/kerlos-bahaa/)
 
 👤 **BAHATI Marie-Grace**
```

Comment:
- [ ] It will be great to ensure the link to the MIT license is not broken. At the moment, the link is showing a 404 error page. Kindly improve on this.
- [ ] It will be great to ensure the link to the issues page is not broken. At the moment, the link is showing a 404 error page. Kindly also improve on this.